src/serviceWorker.js

```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```

Comment:
Can you learn what this file is used for and why you think it should be part of the application.

Reply:
Joe looked into this and said: service worker is a js file that sits with the browser and it's like a middle man between the browser and the network. Its used mainly for caching assets for offline browsing and such but it can also be used for push notifications and what not. It is something cool that definitely makes the app better but I don't think we need it

Reply:
Good.